packages/block-library/src/tag-cloud/style.scss

```diff
@@ -18,4 +18,13 @@
 		margin-left: 5px;
 		text-decoration: none;
 	}
+
+	&.is-style-pill a {
+		border: 1px solid currentColor;
+		font-size: var(--wp--preset--font-size--normal) !important; // !important Needed to override the inline styles.
```

Comment:
The [normal font size has been depreciated](https://github.com/WordPress/gutenberg/pull/37381), so we might want to switch to "Medium" here? 

.... but if we don't specify a font size maybe we're better off? Then it should inherit the default size.

Reply:
The problem is if we don't specify a size then the font sizes are determined by how often the tag is used

Reply:
`unset` will do that for you